# GÃ©rer les erreurs et les exceptions

try:
    number = int(input("Entrez un nombre:"))

    print("Le nombre est :", number)
except:
    print("Vous n'avez pas saisi un nombre")